image_provisioner/playbooks/build_ami.yaml

```diff
@@ -0,0 +1,77 @@
+- hosts: localhost
+  connection: local
+  become: no
+  tasks:
+
+  - name: launch temporary instance
+    connection: local
+    become: no
+    ec2:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      assign_public_ip: yes
+      region: "{{ region }}"
+      key_name: "{{ key_name }}"
+      group_id: "{{ group_id }}"
+      instance_type: "{{ instance_type }}"
+      vpc_subnet_id: "{{ vpc_subnet_id }}"
+      image: "{{ base_image }}"
+      volumes:
+        - device_name: /dev/sdb
+          volume_type: io1
+          volume_size: 30
```

Comment:
We'd like, if possible, that this work be used by both teams so that there we're both using the same AMIs.  Hopefully account for whatever unique case you might have (disk space...any others?).  Is this reasonable?